Makefile

```diff
@@ -5,12 +5,21 @@ AWS_DISTRIBUTE=
 BUILDDIR=.build
 MAINTAINER_EMAIL="contact@gardenlinux.io"
 
+ifneq ($(wildcard .packages),)
+LOCAL_PKGS=.packages
```

Comment:
Please don't use a "hidden" directory. People might not see it and it's contents and will be surprised.

Also, please create that directory and add a README.

Reply:
Can't we just re-use `./packages` for that?

Reply:
I would use `./local_packages`.

Reply:
For comparison: the Debian tools use `./localdeb`.

Reply:
I think this discussion is unrelated to this change.
The output of the local packages builds is currently```.packages```.
[see](https://github.com/gardenlinux/gardenlinux/blob/48abbeddee267cb829b29802a5b9c0ee83866c41/packages/Makefile#L1)

However, I second @waldiTM argument not to use a hidden directory..

Reply:
personally, I find a top-level directory for this a bit polluting, so would vote for `./packages/localdeb`, but I'll refrain from further bikeshedding here.

I agree with @waldiTM that a hidden directory is probably not a good idea